AppController/djinn.rb

```diff
@@ -4035,14 +4032,11 @@ def copy_encryption_keys(dest_node)
 
     cloud_keys_dir = File.expand_path("#{APPSCALE_CONFIG_DIR}/keys/cloud1")
     make_dir = "mkdir -p #{cloud_keys_dir}"
-
-    cloud_private_key = "#{cloud_keys_dir}/mykey.pem"
-    cloud_cert = "#{cloud_keys_dir}/mycert.pem"
+    change_permissions = "chmod 600 #{APPSCALE_CONFIG_DIR}/ssh.key}"
 
     HelperFunctions.run_remote_command(ip, make_dir, ssh_key, NO_OUTPUT)
-    HelperFunctions.scp_file(ssh_key, ssh_key, ip, ssh_key)
-    HelperFunctions.scp_file(cloud_private_key, cloud_private_key, ip, ssh_key)
-    HelperFunctions.scp_file(cloud_cert, cloud_cert, ip, ssh_key)
+    HelperFunctions.scp_file(ssh_key, "#{APPSCALE_CONFIG_DIR}/ssh.key", ip, ssh_key)
+    HelperFunctions.run_remote_command(ip, change_permissions, ssh_key, NO_OUTPUT)
```

Comment:
I don't think so. It is possibly redundant since the AC will set the permission accordingly when it starts. Should I remove it?